### ISSUE TITLE
FORUM REQUEST

### ISSUE THREAD

**Issue by ignacio-rocco** [vlfeat/matconvnet#708]
I see many "issues" on this Github are actually questions. That's why I think there should be a separate user Forum where typical questions can be addressed, and discussion encouraged, while the Issues board of Github should be left for real development issues.

**Comment by lenck** [comment]
Yeah, good point... We have google groups, but still people tend to post questions for help here... I was thinking maybe to start a subtopic on StackOverflow - as the voting system would help to sort out the most common problems?
Would be grateful for any idea :)

**Comment by lenck** [comment]
Hmm, just found out that I don't have enough reputation to start a tag on StackOverflow... So this plan gets a bit more complicated ;)

**Comment by ignacio-rocco** [comment]
Well maybe Google Groups is fine, but I think it lacks a bit the structure forums usually have (by adding sub-forums for each particular topic (ie: getting started / simple-nn / dag / mex, or some other partitioning). I feel it would be a great way to discuss ideas and share code which might not justify to be included in the official release. Furthermore, you can have these sticky posts which tend to solve for people's most usual questions. 

Maybe it's not needed, and Google Groups is just fine, it's just an idea.

**Comment by lenck** [comment]
Yeah, that's a good point. But for a forum we do not have the means to be able to moderate it in a reasonable way :(
What I like about Stack Overflow is the fact that you can vote for the questions - inherently sorting out what are the common problems and also pick the best answers...

**Comment by Roboball** [comment]
Please do simple examples to every function. That will avoid many problems.

**Comment by rlayne** [comment]
Personally I'd recommend something like gitter.im which I've found to be excellent when collaborating on remote projects/discovering bugs and whatnot. I'm not sure what degree of archivalness it offers, but certainly you could consider it a powerful cue for what needs to be added to the wiki/faq.

**Comment by hdm29** [comment]
hey there, 
i'm having problem where to start on matconvnet. is there any tutorial ? i read the documentation but it's not enough for dummies like me. a complete example would be the best. Now i am trying to use vl_imreadjpeg.
